README.rst

```diff
@@ -991,6 +991,11 @@ routes_formatter
   format usable in the documentation when using the `api!` keyword. By
   default, the ``Apipie::RoutesFormatter`` is used.
 
+swagger_op_id_formatter
```

Comment:
looks fine but got me wondering 'what is an op id'

Reply:
It's just what apipie uses to identify certain operations in the openapie spec.  Specifically, these are `path` or `method` identifiers (paths are routes), and methods are input param groupings.